airbyte-cdk/python/airbyte_cdk/entrypoint.py

```diff
@@ -176,6 +176,13 @@ def set_up_secret_filter(config: TConfig, connection_specification: Mapping[str,
     def airbyte_message_to_string(airbyte_message: AirbyteMessage) -> Any:
         return airbyte_message.json(exclude_unset=True)
 
+    @classmethod
+    def extract_state(cls, args: List[str]) -> Optional[Any]:
```

Comment:
we can use this from the main.py to create different streams when running in full refresh or incrementally. [example for stripe](https://github.com/airbytehq/airbyte/pull/31327/files#diff-ab40a7ecb7009e3b038d40cda9e9a3dbe43725152dff44fa09a3c91bc8ea95c6R12-R16)